src/NCronJob/Registry/JobRegistry.cs

```diff
@@ -26,13 +26,14 @@ public void Add(JobDefinition jobDefinition)
     {
         AssertNoDuplicateJobNames(jobDefinition.CustomName);
 
-        var isTypeUpdate = allJobs.Any(j => j.JobFullName == jobDefinition.JobFullName);
-        if (isTypeUpdate)
+        if (!allJobs.Add(jobDefinition))
```

Comment:
Maybe reword the exception message thrown by ` AssertNoDuplicateJobNames` so that it better aligns with the wording below".

Reply:
Thanks - done